tests/rules/javascript/test.opendialog_remote_uri.js

```diff
@@ -0,0 +1,41 @@
+import { VALIDATION_WARNING } from 'const';
+import JavaScriptScanner from 'scanners/javascript';
+import * as messages from 'messages';
+
+
+describe('opendialog_remote_uri', () => {
+
+  it('should warn on remote uris passed to openDialog', () => {
+    var code = `foo.openDialog("https://foo.com/bar/");
+                foo.openDialog("http://foo.com/bar/");
+                foo.openDialog("ftps://foo.com/bar/");
+                foo.openDialog("ftp://foo.com/bar/");
+                foo.openDialog("//foo.com/bar/");
+                foo.openDialog("data:whatever");`;
+    var jsScanner = new JavaScriptScanner(code, 'badcode.js');
+
+    return jsScanner.scan()
+      .then((validationMessages) => {
+        assert.equal(validationMessages.length, 6);
+        for (let msg of validationMessages) {
```

Comment:
Nitpick here, but I always like full variable names when we can get them, so `message` over `msg` if it's possible.